simulator/mips/t/unit_test.cpp

```diff
@@ -1064,11 +1022,276 @@ TEST_CASE( "MIPS32_instr: lui with 0xfee1")
 
 static auto get_plain_memory_with_data()
 {
-    auto memory = FuncMemory::create_plain_memory(15);
+    auto memory = FuncMemory::create_plain_memory(13);
     memory->write<uint32, Endian::little>( 0xABCD'1234, 0x1000);
```

Comment:
I suggest to be more verbose here and write data by bytes, either with `write` or `memcpy_host_to_guest`.